api/handlers/dispute/model.go

```diff
@@ -488,3 +490,109 @@ func (m *disputeModelReal) GenerateAISummary(disputeID int64, disputeDesc string
 		return
 	}
 }
+
+func (m *disputeModelReal) GetAdminDisputes(searchTerm *string, limit *int, offset *int, sort *models.Sort, filters *[]models.Filter, dateFilter *models.DateFilter) ([]models.AdminDisputeSummariesResponse, error) {
+	logger := utilities.NewLogger().LogWithCaller()
+	var disputes []models.AdminDisputeSummariesResponse = []models.AdminDisputeSummariesResponse{}
+	var queryString string = ""
+	var searchString string = ""
+	var filterString string = ""
+	var dateFilterString string = ""
+	var sortString string = ""
+	var limitString string = ""
+	var offsetString string = ""
```

Comment:
small  nit, but you don't need to include the type definition. you can just go `queryString := ""`